### DISCUSSION TITLE
The more I read the more confused I get

### DISCUSSION THREAD

**Discussion by UserWally** [Guilouz/Klipper-Flsun-Speeder-Pad#81] [General]
I am terribly intimidated by the guide. Not sure what is mandatory and what is for more advanced settings.
After I update the image to 1.2 (Which I just did) is it mandatory to complete the rest of the guide as listed? SSH and all that? Or am I good to go with just the image?

**Comment by UserWally** [comment]
Well I made three attempts to install this version. All three times I got stopped at firmware update for the V400. It says that the needed file for the V400 is not in the Klipper/out directory.
I spent the last three hours going backwards and am currently reimaging to the FlSUN 1,1 so I can at least use my printer.
Maybe I'm old, maybe I'm stupid, but for sure this is overly complicated for an average user.

**Reply by Kaotic3** [reply]
I had this problem as well - the issue I think is you are not correctly selecting the items in the MenuConfig -

I didn't realise but if you look a little closer you have to set more than just the box at the top - I did that, not realising that the other entries are actually different.

You have to select each one, go in, select the one that is in the image, and then move on - then Q and Y - then do the make commands and this will create a klipper/out directory.

**Reply by UserWally** [reply]
OK. I tried again but the only options coming up are for Atmega. No stm32 options are listed. I am at a loss.

**Reply by rpm8200** [reply]
Guys, I agree, it is not plain simple and maybe a bit too much for an average user. 
I try to provide a guide on the whole install process as a YT video. I hope I get it done.
The Firmware update also bugged me.
Yes, you have to do the really really proper settings and you shall not mess up... but even with proper settings I had an issue obviously with my mSD Card. I repeated the process multiple times, it would not flash... until I bought a seperate card for that (only 256MB) and then it worked for me.
But you have a different problem. Maybe it would be useful to share a screenshot of the step you get stuck?

**Reply by UserWally** [reply]
4th time was the charm I finally successfully got through the install. Everything appears to have installed correctly. Did the calibration routine.  All good. Went to do the first layer test print using Cura and I don't know what the heck is happening. When I initiate the print, the head lowers about 50 mm and swings all the way to the left and parks there. Remaining time on a 10 minute print shows 4 days and climbing the longer it sits there. Hot end temp is locked at 150 and will not let me change it after the print is initiated. Cancel appears to do nothing. I must reboot to get the head to rehome. Does any of this lunacy make sense to anyone. I have switched everything off and restarted several times. No change. Hopefully there is a recognizable issue occurring that I can get advice on. I'll try Prusa Slicer tomorrow and see if I get the same actions. I really don't think it's a slicer issue.
I'll fiddle around a few more hours tomorrow but if I have no luck I guess I'll be going back to the Original FLSUN setup.

**Reply by rpm8200** [reply]
Hm. Did you also change the StartCode of Cura? 
Chapter Slice-Side Changes...

**Reply by UserWally** [reply]
Yes I did

**Reply by UserWally** [reply]
OK. Something very weird is going on. I uploaded the test chep cube gcode file that came with the V400 and it acted and printed perfectly normal and came out fine.
So I uninstalled every CURA install and roaming folders from all versions. (Wish I had not) I did a clean install of Cura 5.3.0. Installed the V400 from CURA 5.3.0. Changed the Start and End codes. Sliced a cal cube. Sent it to the printer and the same crazy stuff happened.
Head lowers a bit, swings left and parks there. Nozzle temp locked at 150 despite the 210 setting in the slicer. Please heeeeeeeellllllppplp
I suspected the printer.cfg file, but if that was wonky, wouldn't that also affect the gcode I tested that worked fine?

**Reply by UserWally** [reply]
Problem resolved. I reloaded the printer in 5.3 and added a 2 at the end. This time I did not touch the start or end codes and everything is fine.
Thanks for listening to my venting. It's been quite a journey.
If you have a V400 and you use the printer profile that comes with Cura 5.3.0. LEAVE IT BE!!

**Reply by 808mp5** [reply]
That’s normal behavior with original klipper… and much better. Nozzle parks off to the left away from the build area, heats bed first while preheating the nozzle to 150c. When bed come up to temp then it will heat the nozzle to temp… use the start and end gcodes per the guide.  If you want it to behave like Flsun build then you should still use the guide cura start and end gcode. Go to the macro config and edit your start and end print gcodes(iirc that’s where they are) there.

**Reply by UserWally** [reply]
Duh, OMG. Thank you soooooo much. I have less than a year in 3D printing. All with Marlin.
You have finally gotten my brain down the right path. It all makes sense now. I can see Klipper will grow on me quickly. So many easy to modify choices without recompiling.
Now if I could only get my Creality CR-X hooked up to the pad as a second printer, I would be a happy camper.
